docs/topics/settings.rst

```diff
@@ -275,6 +275,25 @@ project name). This name will be used for the logging too.
 It's automatically populated with your project name when you create your
 project with the :command:`startproject` command.
 
+.. setting:: CHECK_SETTINGS_ENABLED
+
+CHECK_SETTINGS_ENABLED
+----------------------
+
+Default: ``True``
+
+Whether to enable the :class:`~scrapy.extensions.checksettings.CheckSettings`
+extension. Optionally a list of variables to ignore.
+
+.. setting:: CHECK_SETTINGS_IGNORED
+
+CHECK_SETTINGS_IGNORED
+----------------------
+
+Default: ``[]``
+
+List of settings variables to ignore by Check Settings extension.
```

Comment:
Since people usually read settings documentation separately, as opposed to reading the whole documentation, I would also use ``` :class:`~scrapy.extensions.checksettings.CheckSettings` ``` here.